src/nxdoc/nuxeo-add-ons/nuxeo-mail-amazon-ses.md

```diff
@@ -0,0 +1,64 @@
+description: The Nuxeo Mail Amazon SES addon allows you to send emails via AWS SES. To do so, it adds a Nuxeo MailSender implementation specific to Amazon SES which overrides the default SMTPMailSender and leverages the Nuxeo MailService.
+tree_item_index: 350
+---
+
+The [Nuxeo Mail Amazon SES](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-mail-amazon-ses) addon allows you to send emails via [AWS SES](https://docs.aws.amazon.com/ses). To do so, it adds a Nuxeo `MailSender` implementation specific to Amazon SES which overrides the `"default"` `SMTPMailSender` and leverages the Nuxeo `MailService`.
+
+## Before You Start
+
+You should be familiar with Amazon AWS and in possession of your credentials and region.
+You should also be familiar with [SES reputation](https://docs.aws.amazon.com/ses/latest/dg/monitor-sender-reputation.html) and [identity verification](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html).
+
+{{#> callout type='tip'}}
+Knowing how to use the [SES Simulator](https://docs.aws.amazon.com/ses/latest/dg/send-an-email-from-console.html) and being aware of the [SES sandbox](https://docs.aws.amazon.com/ses/latest/dg/request-production-access.html) limitations is a nice-to-have.
+{{/callout}}
+
+## Installation
+
+{{{multiexcerpt 'MP-installation-easy' page='Generic Multi-Excerpts'}}}
+
+{{#> callout type='info' }}
+The `ses` template leverages the `aws` template and its configuration (see below).
+{{/callout}}
+
+## Nuxeo Configuration
+
+To configure the package, you will need to provide values for some configuration parameters defined in the `aws` template.
+You can configure the package using the `nuxeo.conf` properties described below.
+
+### Specifying Your AWS Credentials and Region
+
+#### Default AWS Credentials and Region
+
+By default, Nuxeo will get your `"default"` [configuration](../amazon-s3-online-storage/#nuxeo-configuration) by leveraging the following properties in `nuxeo.conf`:
+
+{{{multiexcerpt 'aws-config-credentials' page='amazon-s3-online-storage'}}}
+
+However, if those are not set, the fallback mechanism will use the [DefaultAWSCredentialsProviderChain](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html). So it will first look for the following envs (and so on with the fallbacks):
+```
+AWS_ACCESS_KEY_ID=your_AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY=your_AWS_SECRET_ACCESS_KEY
+AWS_REGION=your_AWS_REGION
+```
+
+#### Specific AWS Credentials for SES
+
+You can also use specific AWS Credentials with SES.
+To do so, you need to contribute a configuration on the following extension:
+
+```xml
+<extension target="org.nuxeo.runtime.aws.AWSConfigurationService" point="configuration">
+  <configuration id="mySESspecificConfigId">
+    <accessKeyId>your_AWS_ACCESS_KEY_ID</accessKeyId>
+    <secretKey>your_AWS_SECRET_ACCESS_KEY</secretKey>
+    <region>your_AWS_REGION</region>
+  </configuration>
+</extension>
+```
```

Comment:
Here I wanted a link to the nuxeo-explorer showing the configuration endpoint but:
1 - 2023 is not available yet
2 - 2021 seems to have problems on latest `HF`s (I slacked it)